cpp/src/text/minhash.cu

```diff
@@ -174,24 +193,45 @@ std::unique_ptr<cudf::column> minhash(cudf::strings_column_view const& input,
 }  // namespace detail
 
 std::unique_ptr<cudf::column> minhash(cudf::strings_column_view const& input,
-                                      cudf::numeric_scalar<cudf::hash_value_type> seed,
+                                      cudf::numeric_scalar<uint32_t> seed,
                                       cudf::size_type width,
-                                      cudf::hash_id hash_function,
                                       rmm::mr::device_memory_resource* mr)
```

Comment:
Is this API-breaking change a good time to introduce public streams, too? (Totally fine if the answer is no / not yet.)